books/models.py

```diff
@@ -33,3 +35,28 @@ class Book(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class CustomLog(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    message = models.CharField(max_length=500)
+    performed_by = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.message
+
+
+@receiver(post_delete, sender=Author)
```

Comment:
Can we please move all your signal handler in separate file "signals.py". The "models.py" should only contain database models.

Reply:
done